src/index.ts

```
@@ -4,10 +4,11 @@ import { McpServer } from "@modelcontextprotocol/sdk/server/mcp.js"
 import { StdioServerTransport } from "@modelcontextprotocol/sdk/server/stdio.js"
 import { registerGeocodeTool } from "./tools/geocode.js"
 import { registerReverseGeocodeTool } from "./tools/reverseGeocode.js"
+import packageJson from '../package.json' with { type: 'json' };
```

ESLint

[error] Extra semicolon.

(semi)